src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -763,6 +763,22 @@ public void contributionsLatestCountDensityFilteredByTagChange() {
         deltaPercentage);
   }
 
+  @Test
+  public void contributionsLatestCountGroupByBoundary() {
```

Comment:
Should we implement the same for POST?

Reply:
IMHO it's overkill, since we get the support of POST "for free" by spring (as long as the appropriate spring annotation is set correctly) and would always produce the same result as the respective GET request. Or what do you think?